### ISSUE TITLE
Add channel reciprocate option

### ISSUE THREAD

**Issue by elkimek** [LN-Zap/zap-desktop#2586]
Add channel reciprocate option under Manage Channels

## Detailed Description

Add an option to reciprocate channel with adjustable capacity under Channels -> Manage to easily "open back" to the 2nd peer without the need to look up for pubkey and do it manually. 

## Context

I feel it would tempt me to reciprocate :D
Moar channels, moar liquidity :)

## Possible Implementation

The best would be to put a second button next to red "Close Channel" imo.

## Your Environment

- Zap version:
- Operating System and version:

**Comment by korhaliv** [comment]
Thanks for the suggestion @elkimek. I can see this improvement as being quite useful!

**Comment by bolatovumar** [comment]
I wouldn't mind taking this on. How do you go about these requests that have "needs design" as status? Do I just come up with something simple to start with, submit that and iterate from there?

**Comment by mrfelton** [comment]
@bolatovumar yes I'd suggest building out and focusing on the functionality and add a simple UI that does the job and fits in with our existing design patters as much as possible. Getting a working MVP will spur on additional conversation and enable us with a starting point to further iterate from.

**Comment by bolatovumar** [comment]
@mrfelton as I understand it in order for others to open a channel with me first (so I can reciprocate from the UI) I need to run an LND instance of my own rather than relying on Neutrino, right? Or does Zap allow others to open a channel with out of the box?

I'm currently spinning up a BTCPay VM so I can have LND running on testnet which I can connect to through Zap. This sounds like it should work, right?

**Comment by mrfelton** [comment]
generally speaking that is correct. There is one exception to this though - if you connect to a peer first, then that peer should be able to open a channel back to you - even to our neutrino nodes. Obviously though in the real world it is unlikely that people would be opening channels directly to neutrino nodes since it requires an outgoing connection to be established from the neutrino node first.

For remote nodes there should be less of an issue since the likelyhood is that the remote nodes are advertising themselves to the network and therefore opening channels to those nodes should be more straightforward.

**Comment by bolatovumar** [comment]
I got my BTCPay Server with LND up and running so I should be able to start working on this now.

**Comment by JimmyMow** [comment]
@bolatovumar I think maybe we should discuss more in Slack and gather consensus. I wouldn't want anyone working on something that won't get merged or that doesn't have consensus on what it will look like when merged, especially when there are lower hanging issues to knock out in the meantime

**Comment by bolatovumar** [comment]
@JimmyMow sure, I haven't really started on this anyway. desktop-dev channel on Slack?
In the meantime your reckon any of the issues marked with **v0.6.0-beta** should be good to work on?
